content/en/docs/languages/java/automatic/spring-boot.md

```diff
@@ -165,7 +165,16 @@ IDE.
 
 The OpenTelemetry Starter supports all the
 [SDK Autoconfiguration](/docs/languages/java/automatic/configuration/#sdk-autoconfiguration)
-(since 2.2.0). You can set properties in the `application.properties` or the
+(since 2.2.0). 
+
+{{% alert title="Attention" color="warning" %}}
```

Comment:
While "Attention" would make sense in some languages other than English, I'd suggest using either "Caution", "Important", "Note", or simply "Warning"; e.g.:

```suggestion
{{% alert title="Important" color="warning" %}}
```